cos-fleet-catalog-connectors-it/azure/eventhubs-it/src/test/groovy/org/bf2/cos/connector/camel/it/ConnectorIT.groovy

```diff
@@ -13,14 +13,14 @@ import spock.lang.IgnoreIf
 import java.util.concurrent.LinkedBlockingQueue
 import java.util.concurrent.TimeUnit
 
+@IgnoreIf({
+    !hasEnv('AZURE_EVENTHUB_NAME'      ) ||
```

Comment:
Looks like a `hasEnvs` could come in handy. :P

Reply:
will add in a followup pr, also we may create a custom annotation for that